ipinfo/cmd_init.go

```diff
@@ -0,0 +1,301 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"os"
+	"os/exec"
+	"runtime"
+	"time"
+
+	"github.com/ipinfo/cli/lib/complete"
+	"github.com/ipinfo/cli/lib/complete/predict"
+	"github.com/spf13/pflag"
+	"golang.org/x/term"
+)
+
+var completionsInit = &complete.Command{
+	Flags: map[string]complete.Predictor{
+		"-t":         predict.Nothing,
+		"--token":    predict.Nothing,
+		"--no-check": predict.Nothing,
+		"-h":         predict.Nothing,
+		"--help":     predict.Nothing,
+	},
+}
+
+func printHelpInit() {
+	fmt.Printf(
+		`Usage: %s init [<opts>] [<token>]
+Examples:
+	# Login command with token flag.
+	$ init --token <token>
+
+	#Authentication without token flag.
```

Comment:
Spacing is messed up - don't use the 'real' tab in these help messages, use the raw space character.

```suggestion
    # Authentication without token flag.
```